hangulime.js

```diff
@@ -1011,6 +1011,9 @@ var num = 4;
 var stackPressedKeys = new FixedStack(num);
 var stackValidJamos = new FixedStack(num);
 
+// Controls whether or not "command" has been pressed for copying, pasting, or cutting (MacOS)
+var isCmdPressed = false;
```

Comment:
nitpick: Had a look at this again and noted that this boolean isn't used anywhere, it should be removed

Reply:
I thought that it was used in the code.

Reply:
The code is setting a value for `isCmdPressed`, but the `isCmdPressed` variable itself isn't being used anywhere else for decision making, not even for logging. It currently has no purpose being in the code